extensions-core/kafka-extraction-namespace/src/test/java/io/druid/query/lookup/TestKafkaExtractionCluster.java

```diff
@@ -261,18 +263,14 @@ public void configure(Binder binder)
     consumerProperties.put("zookeeper.session.timeout.ms", "10000");
     consumerProperties.put("zookeeper.sync.time.ms", "200");
 
-    final KafkaLookupExtractorFactory kafkaLookupExtractorFactory = new KafkaLookupExtractorFactory(
-        null,
-        topicName,
-        consumerProperties
-    );
-
     factory = (KafkaLookupExtractorFactory) mapper.readValue(
-        mapper.writeValueAsString(kafkaLookupExtractorFactory),
-        LookupExtractorFactory.class
+          mapper.writeValueAsString(new KafkaLookupExtractorFactory(
+              null,
+              topicName,
+              consumerProperties, 0, false, NOOP_CONNECTION_FACTORY
+          )), LookupExtractorFactory.class
     );
-    Assert.assertEquals(kafkaLookupExtractorFactory.getKafkaTopic(), factory.getKafkaTopic());
-    Assert.assertEquals(kafkaLookupExtractorFactory.getKafkaProperties(), factory.getKafkaProperties());
```

Comment:
Asserting that serialization and deserialization work correctly was already present in `KafkaLookupExtractorFactoryTest`, so it was removed from here.